src/components/ServiceRequestForm.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
```

Comment:
this value is going to need to come from an ajax request instead of importing because @danfox01 and webservices will need to update these files.

checkout how to do this https://www.robinwieruch.de/react-hooks-fetch-data/

Reply:
implemented react hooks to fetch data

Reply:
data coming from json file instead now.